### PR TITLE
Update vpn.mozilla.org links on /whatsnew to point to /products/vpn/ (Fixes #9942)

### LINKED ISSUE #9942: Update vpn.mozilla.org links to point to /products/vpn/
## Description

Once the VPN landing page is live on www, we should update bedrock's existing `vpn.mozill.org` links to point to `/products/vpn/`. This should be done before the VPN team redirects the old landing page to www (which will be [TBD] weeks after the new landing page goes live).

There are [only a few VPN links](https://github.com/mozilla/bedrock/search?q=vpn.mozilla.org) in bedrock currently, however the highest traffic ones are from [/whatsnew pages](https://github.com/mozilla/bedrock/blob/9d55ac4b758223bc75c7ecff1599d7177404d868/bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx85.html#L32). 

When we change those links, we should remove the utm parameters (since utms on internal links create new sessions in GA). When we do that, we'll still be able to see how many clicks those links get, and how many of those visitors go on to convert a goal on the new VPN landing page. However, without the utms  in place the VPN product team would not be able to attribute traffic from the /whatsnew page directly further down the funnel (if I understand things right).

Maybe the above is ok (?) and enough for what marketing need to understand as far as /whatsnew goes. Or maybe we need to go a step further and make referrals from /whatsnew directly attributable. Using some other unique referrer (something like the `entrypoint_experiment` and `entrypoint_variation` params) may be a possible solution here.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- http://localhost:8000/en-US/firefox/84.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/85.0/whatsnew/all/

## Issue / Bugzilla link
#9942

## Testing
- [ ] No stray links to vpn.mozilla.org remain in bedrock.

### CODE REVIEW

**Comment by alexgibson** [comment]
This change needs to be live in prod before Wed 10th March, which is the data when the old landing page is due to be redirected to www.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1 class="wnp-main-title">Firefox protects your browsing. <strong>Mozilla VPN</
       <p class="wnp-main-tagline">Mozilla VPN (Virtual Private Network) protects your entire internet connection on your computer, your tablet, and even your phone. For even more security, it comes with a 30-day money back guarantee.</p>
 
       <div class="wnp-main-cta">
-        <a href="https://vpn.mozilla.org/?{{ utm_params }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">Get Mozilla VPN Now</a>
+        <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">Get Mozilla VPN Now</a>
```
Comment (maxxcrawford) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html:32]: Because the page is now internal, it's okay to drop the `utm_params`, right?

**Review by maxxcrawford — APPROVED** [review]
I confirmed the link works on both pages! Good to go. 🏄 

Quick note – I searched across the code base for all instances of `vpn.mozilla.org` and found a  few. Should this be in a follow-up, low-priority bug? (LMK and I can file)

**Comment by alexgibson** [comment]
> Quick note – I searched across the code base for all instances of vpn.mozilla.org and found a few. Should this be in a follow-up, low-priority bug? (LMK and I can file)

I should take care of those here, what did you find?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html:32]: Yes we should drop utms now these are same site links 👍

Comment (maxxcrawford) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx84.html:32]: > > Quick note – I searched across the code base for all instances of vpn.mozilla.org and found a few. Should this be in a follow-up, low-priority bug? (LMK and I can file)
> 
> I should take care of those here, what did you find?

I found the following that may apply: 
- [ ] [web-eligible-sites.html](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/security/templates/security/bug-bounty/web-eligible-sites.html#L251)
- [ ] [fxa-utm-referral.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/fxa-utm-referral.js#L23)
- [ ] [mozilla-fxa-product-button.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/mozilla-fxa-product-button.js#L21)

There were a few other links ([1](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L69), [2](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42), [3](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42)) but I don't think they apply.

**Comment by maxxcrawford** [comment]
> > Quick note – I searched across the code base for all instances of vpn.mozilla.org and found a few. Should this be in a follow-up, low-priority bug? (LMK and I can file)
> 
> I should take care of those here, what did you find?

I found the following that may apply: 
- [ ] [web-eligible-sites.html](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/security/templates/security/bug-bounty/web-eligible-sites.html#L251)
- [ ] [fxa-utm-referral.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/fxa-utm-referral.js#L23)
- [ ] [mozilla-fxa-product-button.js](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/media/js/base/mozilla-fxa-product-button.js#L21)

There were a few other links ([1](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L69), [2](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42), [3](https://github.com/alexgibson/bedrock/blob/vpn-whatsnew-links/bedrock/products/tests/test_helper_misc.py#L42)) but I don't think they apply.

**Comment by alexgibson** [comment]
We only need cover hyperlinks here, so I think all these are OK and don't need changing here. Thanks for checking! 💯

**Pull request merged**